appliance/postgresql/main.go

```diff
@@ -43,7 +44,7 @@ func main() {
 	}
 	inst := &discoverd.Instance{
 		Addr: ":5432",
-		Meta: map[string]string{"POSTGRES_ID": id},
+		Meta: map[string]string{"DB_ID": id},
```

Comment:
Yes I think so, though doing it live may prove to be somewhat difficult.
If we want the `postgres` deployment strategy to be made generic these variables would either need to be standardised or the prefix stored in controller/job or discoverd metadata.
Will think it over.